### DISCUSSION TITLE
Backups

### DISCUSSION THREAD

**Discussion by JustinGuese** [mysticaltech/terraform-hcloud-kube-hetzner#585] [Q&A]
Touching on the old topic again, because we did not yet find a solution... Anyone got backups to work with the classic Kasten K10s, Velero?
We are struggling with the volumes... The configs and everything works, but volumes not really. In earlier versions of Velero it worked with restic, but something changed and it does not work anymore... Can someone point me in the right direction?

**Comment by JustinGuese** [comment]
We would be up for sponsoring this integration as well

**Comment by otavio** [comment]
We use Longhorn for the storage and it has support for volume backups bult-in.

**Reply by aDingil** [reply]
https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/discussions/765#discussioncomment-5949629
